ghcide/src/Development/IDE/Spans/AtPoint.hs

```diff
@@ -59,6 +60,7 @@ import           Data.List.Extra                      (dropEnd1, nubOrd)
 
 import           Data.Version                         (showVersion)
 import           Development.IDE.Types.Shake          (WithHieDb)
+import           GHC                                  (getInstancesForType)
```

Comment:
Can we get it from Compat?

Reply:
Since in general we import ghc stuff from `Development.IDE.GHC.Compat`

Reply:
so I should just export it in `Development.IDE.GHC.Compat`? Why is this needed? It seems to be available in this module in all the ghc versions supported by hls

Reply:
I'm unsure of our policy here, wdyt @wz1000 ?